backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/pporder/PP_Order_Candidate_StepDef.java

```diff
@@ -668,4 +702,17 @@ private String getPPOrderCandidateContextForProductId(@NonNull final ProductId p
 
 		return "Current PP_Order_Candidates available for M_Product_ID:\n\n" + messageBuilder;
 	}
+
+	@And("after not more than {int}s, no PP_Order_Candidates are found for Issue_HU_ID: {string}")
+	public void validatePP_Order_Candidate(
+			final int timeoutSec,
+			final String issueHuIdIdentifier) throws InterruptedException
```

Comment:
"IdIdentifier" sounds like a pleonasm. I suggest you name this variable "issueHUIdentifier"

Reply:
done.